src/script/error/BaseError.ts

```diff
@@ -27,11 +27,9 @@ export class BaseError extends Error {
   type: BASE_ERROR_TYPE | string;
 
   constructor(type: BASE_ERROR_TYPE | string, message: string) {
-    super();
+    super(message);
 
     this.type = type;
-    this.message = message;
-    this.stack = new Error().stack;
```

Comment:
rewritting the stack was the problem here. We were adding one more element on top of the stack which prevented the browser from showing correctly the error message